### PR TITLE
Just updating the bundle installation as well as other bundles.

### PR DESCRIPTION
**Pull request by yourwebmaker**

### CODE REVIEW

File: Resources/doc/index.rst [hunk 1]
```diff
@@ -41,7 +41,7 @@ following:
     {
         $bundles = array(
             //...
-            new Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle(),
+            $bundles[] = new Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle();
```
Comment (xabbuh) [inline Resources/doc/index.rst]: What's the reason for this change? This now is not even valid PHP code anymore.

Comment (yourwebmaker) [inline Resources/doc/index.rst]: sorry... I was trying to standardize bundle installation, but code was for a dev/test bundle. Sorry.
